### PR TITLE
Add http-server with npm script

### PR DESCRIPTION
**Pull request by MattMS**
Add http-server as a dev dependency and a npm script to start a local server.

### CODE REVIEW

**Comment by CrossEye** [comment]
I have no objection, I suppose.  I do this all the time, but I have http-server installed globally, as I use it all over the place, so I've never thought about including.

:herb:

**Comment by buzzdecafe** [comment]
yes, this makes sense. I have a local apache i use, but no reason not to do this. :cow2:

File: package.json [hunk 1]
```diff
@@ -13,8 +13,12 @@
     "gitbook-cli": "0.3.x",
     "handlebars": "2.x.x",
     "highlight.js": "8.x.x",
+    "http-server": "^0.8.5",
```
Comment (davidchambers) [inline package.json]: Nit: Let's use `"0.8.x"` to be consistent.

Comment (MattMS) [inline package.json]: I simply did `npm i http-server`.

Have updated now anyway.

File: package.json [hunk 2]
```diff
@@ -13,8 +13,12 @@
     "gitbook-cli": "0.3.x",
     "handlebars": "2.x.x",
     "highlight.js": "8.x.x",
+    "http-server": "^0.8.x",
```
Comment (davidchambers) [inline package.json]: Let's remove the `^`.

**Comment by MattMS** [comment]
Just tested and it does default to the current folder, so I've removed the `./`.

File: package.json (hunk 2 above)
Comment (MattMS) [inline package.json]: Done!

**Comment by CrossEye** [comment]
This looks good to me.  Could you squash the commits?

**Comment by MattMS** [comment]
I haven't used squashed commits before and it doesn't look like I did it right (it just created 2 new commits).

**Comment by davidchambers** [comment]
@MattMS, is https://github.com/ramda/ramda/pull/1571#issuecomment-168410352 instructive?

**Comment by MattMS** [comment]
Thanks @davidchambers 

I was reluctant to force push, but it seemed to work. :smile:

**Comment by CrossEye** [comment]
Thanks @davidchambers.

My git-fu has finally gotten strong enough that I can do most of these tasks myself.  Explaining them to someone else is still beyond me.

And thanks @MattMS for the PR!

**Comment by MattMS** [comment]
Happy to join the crew! :smile:

**Comment by davidchambers** [comment]
Thanks for the contribution!

**Pull request merged**